vertical-pod-autoscaler/pkg/admission-controller/resource/vpa/handler.go

```diff
@@ -137,6 +137,28 @@ func ValidateVPA(vpa *vpa_types.VerticalPodAutoscaler, isCreate bool) error {
 			if policy.ContainerName == "" {
 				return fmt.Errorf("containerPolicies.ContainerName is required")
 			}
+
+			// check that perVPA is on if being used
+			if err := validatePerVPAFeatureFlag(&policy); err != nil {
+				return err
+			}
+
+			// Validate OOMBumpUpRatio
+			if policy.OOMBumpUpRatio != nil {
+				ratio := float64(policy.OOMBumpUpRatio.MilliValue()) / 1000.0
+				if ratio < 1.0 {
+					return fmt.Errorf("OOMBumpUpRatio must be greater than or equal to 1.0, got %v", ratio)
+				}
+			}
+
+			// Validate OOMMinBumpUp
+			if policy.OOMMinBumpUp != nil {
+				minBump := policy.OOMMinBumpUp.Value()
+				if minBump < 0 {
+					return fmt.Errorf("OOMMinBumpUp must be greater than or equal to 0, got %v bytes", minBump)
```

Comment:
I think this error should be `oomMinBumpUp`, since we want to let the user know the JSON type that is invalid